### PR TITLE
Update contentTypeParser.js

### PR DESCRIPTION
**Pull request by RajPabnani03**
#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by Uzlopak** [comment]
No.

File: lib/contentTypeParser.js [hunk 1]
```diff
@@ -147,8 +147,9 @@ ContentTypeParser.prototype.remove = function (contentType) {
 
   const idx = parsers.findIndex(ct => ct.toString() === contentType.toString())
 
-  if (idx > -1) {
+  if (idx > -1 && idx != 'NULL') {
```
Comment (climba03003) [inline lib/contentTypeParser.js:150]: I don't know what are you doing. `idx` will never be a string `NULL`.

File: lib/contentTypeParser.js [hunk 2]
```diff
@@ -147,8 +147,9 @@ ContentTypeParser.prototype.remove = function (contentType) {
...
     parsers.splice(idx, 1)
+	  through new error("Value cannot be NULL");
```
Comment (climba03003) [inline lib/contentTypeParser.js:152]: `through` is not even a proper syntax in JavaScript.

**Review by climba03003 — COMMENTED** [review]
I would recommend to ban this user.
From seeing it's profile. He is sending spam PR to multiple project.
e.g. https://github.com/freeCodeCamp/freeCodeCamp/pull/48197

**Pull request closed without merging**